### ISSUE TITLE
Output of nomml.py

### ISSUE THREAD

**Issue by shenranwang** [GigaMidiDataset/The-GigaMIDI-dataset-with-loops-and-expressive-music-performance-detection#2]
Hi, I tried running the nomml.py script on my MIDI dataset, but the output seems to only contain the median_metric_depth attribute. Is that supposed to be the NOMML metric? What other attributes should be part of the json output, and how could I figure out which MIDI is expressive or non-expressive?

**Comment by punchline10** [comment]
Yes, that's the same as NOMML. Please check our latest info below, as this repository was meant to be temporary during the anonymous peer-review process of the academic paper. Based on our experiments, you can use levels between 0 and 11 as non-expressive and level 12 as expressive MIDI tracks. Note that it works at the track-level, not the file-level.

We are excited to announce the release of version 1.0 of the GigaMIDI dataset (released on February 7th, 2025). We are actively refining the dataset to improve its usability, adding new musical features, and expanding it with additional subsets. Stay tuned for more exciting updates coming throughout 2025!

TISMIR journal paper link: https://lnkd.in/gnkzuvhq 

Hugging Face Hub: https://lnkd.in/gFG5CcFx 

GitHub Repository: https://lnkd.in/ggS-jGTZ 

Please contact me at keon_maverick@sfu.ca if you have any questions about the GigaMIDI dataset!
